### ISSUE TITLE
Etapa 1 - Display placa

### ISSUE THREAD

**Issue by mariagonthier** [IIC2343/Syllabus-2023-2#22]
Hola!

No logramos entender si para esta entrega se utiliza el display de la placa ¿Hay que conectar los leds o algo?¿ Es solo vía simulación?

Gracias :)

**Comment by frvalenzuela** [comment]
Hola @mariagonthier !

En la CPU existe una señal llamada `dis`, esa estará conectada directamente al display de basys3. Lo que se espera es que a esa señal se conecte lo que sale de los registros para ver el resultado del mismo en alguno de los registros.

Saludos!

**Comment by jlizamajara** [comment]
Hola! Se debe conectar solo el registro donde se muestra el resultado de la multiplicación?

**Comment by frvalenzuela** [comment]
Hola @jlizamajara !

Si bien con eso basta, recomiendo conectar ambos registros usando los primeros 8 bits menos significativos de cada uno.

Saludos!

**Comment by AlvaroSotomayor** [comment]
Hola! tengo una duda respecto al display. Si bien el multiplicador debe ser capaz de mostrar una solucion de 16bits maximo, anteriormente se le dijo @jlizamajara que podia agarrar los 8 bits menos significativos de cada registro, pero al hacer eso ¿no estariamos limitando la capacidad del display de mostrar numeros superiores a 8 bits? ya que la multiplicacion con un numero de 16 bits como resultado no se mostraria correctamente.

**Comment by Geeermy** [comment]
@AlvaroSotomayor Sí, se estarían limitando. Si quieren pueden conectar el registro A completo para poder mostrar los 16 bits, pero se les sugiere que conecten los 8 bits menos significativos de A y B porque dicha configuración les servirá en las siguientes etapas para poder ver cómo se realiza la ejecución de su implementación con distintas instrucciones.

Pueden preocuparse de mostrar multiplicaciones cuyo resultado pueda ser representado con 8 bits para demostrar que su multiplicación funciona, eso no será problema.

¡Saludos! :)
